x/ibc/03-connection/client/cli/tx.go

```diff
@@ -207,7 +206,7 @@ $ %s tx ibc connection open-ack [connection-id] [path/to/proof_try.json] [versio
 			version := args[4]
 
 			msg := types.NewMsgConnectionOpenAck(
-				connectionID, proofTry, proofHeight,
+				connectionID, proofTry, proofTry, proofHeight,
```

Comment:
should the proof always match the `msg.ProofConsensus`? If so we should add that validation on the message. cc: @cwgoes